oslc_prolog/lib/oslc_rdf.pl

```diff
@@ -47,6 +47,25 @@
   must_be(atom, Graph),
   rdf_retractall(IRI, PropertyDefinition, _, Graph).
 
+oslc:marshal_property(IRI, PropertyDefinition, Value, Type, tmp(Graph)) :-
+  ( var(Graph)
+  -> make_temp_graph(Graph)
+  ; true
+  ),
+  oslc:marshal_property(IRI, PropertyDefinition, Value, Type, rdf(Graph)).
+
+oslc:unmarshal_property(IRI, PropertyDefinition, Value, Type, tmp(Graph)) :-
+  ( var(Graph)
+  -> fail
+  ; oslc:unmarshal_property(IRI, PropertyDefinition, Value, Type, rdf(Graph))
+  ).
+
+oslc:delete_property(IRI, PropertyDefinition, tmp(Graph)) :-
+  ( var(Graph)
+  -> make_temp_graph(Graph)
+  ; oslc:delete_property(IRI, PropertyDefinition, rdf(Graph))
+  ).
+
```

Comment:
FEATURE: use `tmp(Graph)` as source/sink to create temporary named graphs. If used in resource handlers, temporary graphs created by a serving thread are automatically garbage collected after response is sent.